src/homeHeader.css

```diff
@@ -1,28 +1,158 @@
-.responsive-header{
+@font-face {
+  font-family: 'SulSans';
+  src:
+    url('./styles/fonts/SulSans-Regular.woff') format('woff');
+  font-weight: 300;
+  font-style: normal;
+  font-display: swap;
+}
+
+.home-header{
   display: none;
   position: sticky;
   flex-direction: column;
-  width: 100vw;
-  height: 80px;
-  background-color: black
+  box-sizing: border-box
 }
 
-.responsive-header__container {
+.home-header__container {
+  height: 80px;
   display: flex;
-  flex-grow: 1;
   justify-content: space-between;
   align-items: center;
   padding: 0;
+  margin: 0 30px;
 }
 
-.responsive-header__logo {
+.home-header__logo {
   display: flex;
   align-items: center;
   cursor: pointer;
 }
 
+.multi-categories-selector__label{
+  padding: 0;
+  margin: 0;
+  height: 80px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  cursor: pointer;
+  border-bottom: 1px solid transparent;
+  transition: border 200ms
+}
+.multi-categories-selector__background-colors-restaurant{
+  width: 40px;
+  height: 40px;
+  background-color: #ea1d2c;
+  border-radius: 100%;
+  display: flex;
+  align-items: center;
+  margin-right: 12px;
+  
+}    
 .multi-categories-selector__label-title {
+  font-weight: bold;
   font-size: 18px;
   color: #717171;
   margin: 0;
-}
\ No newline at end of file
+}
+.icon-marmita--arrow-down:hover {
+  display: block;
+  width: 24px;
+  height: 24px;
+  transform: translateY(2px);
+  margin: 0;
+  color: #ea1d2c;
+}
+
+.search-container{
+  max-width: 469px;
```

Comment:
I think you can remove this width, look, I  removed and the layout became more aligned:

![image](https://user-images.githubusercontent.com/38358239/150128110-27ce98bd-2d8a-48b5-bf35-66232dc757a6.png)


before:
![image](https://user-images.githubusercontent.com/38358239/150128209-c2dd2ba6-57fe-48b3-a4a6-2a48e84880ab.png)

Reply:
thank you <3 removed